### ISSUE TITLE
使用ch_PP-OCRv4_det_cml.yml知识蒸馏训练，数据集和配置文件都和官方的一样，但是训练准确率和召回率很低，最高只有30%。

### ISSUE THREAD

**Issue by jjkgsd** [PaddlePaddle/PaddleOCR#15059]
### 🔎 Search before asking

- [x] I have searched the PaddleOCR [Docs](https://paddlepaddle.github.io/PaddleOCR/) and found no similar bug report.
- [x] I have searched the PaddleOCR [Issues](https://github.com/PaddlePaddle/PaddleOCR/issues) and found no similar bug report.
- [x] I have searched the PaddleOCR [Discussions](https://github.com/PaddlePaddle/PaddleOCR/discussions) and found no similar bug report.

### 🐛 Bug (问题描述)

使用ch_PP-OCRv4_det_cml.yml知识蒸馏训练，数据集和配置文件都和官方的一样，但是训练准确率和召回率很低，最高只有30%，是什么原因呢？
配置文件：
Global:
  debug: false
  use_gpu: true
  epoch_num: 500
  log_smooth_window: 20
  print_batch_step: 20
  save_model_dir: ./output/ch_ppocr_v4_det_icdar_cml
  save_epoch_step: 10
  eval_batch_step:
  - 0
  - 500
  cal_metric_during_train: false
  checkpoints: null
  pretrained_model: 
  save_inference_dir: ./export_models/ch_ppocr_v4_det_cml
  use_visualdl: false
  infer_img: doc/imgs_en/img_10.jpg
  save_res_path: ./checkpoints/det_db/predicts_db.txt
  distributed: true
Architecture:
  name: DistillationModel
  algorithm: Distillation
  model_type: det
  Models:
    Student:
      pretrained: 
      model_type: det
      algorithm: DB
      Transform: null
      Backbone:
        name: PPLCNetV3
        scale: 0.75
        pretrained: false
        det: true
      Neck:
        name: RSEFPN
        out_channels: 96
        shortcut: true
      Head:
        name: DBHead
        k: 50
    Student2: 
      pretrained: 
      model_type: det
      algorithm: DB
      Transform: null
      Backbone:
        name: PPLCNetV3
        scale: 0.75
        pretrained: true
        det: true
      Neck:
        name: RSEFPN
        out_channels: 96
        shortcut: true
      Head:
        name: DBHead
        k: 50
    Teacher:
      pretrained: ./pretrained_models/teacher.pdparams
      freeze_params: true
      return_all_feats: false
      model_type: det
      algorithm: DB
      Backbone:
        name: ResNet_vd
        in_channels: 3
        layers: 50
      Neck:
        name: LKPAN
        out_channels: 256
      Head:
        name: DBHead
        kernel_list:
        - 7
        - 2
        - 2
        k: 50
Loss:
  name: CombinedLoss
  loss_config_list:
  - DistillationDilaDBLoss:
      weight: 1.0
      model_name_pairs:
      - - Student
        - Teacher
      - - Student2
        - Teacher
      key: maps
      balance_loss: true
      main_loss_type: DiceLoss
      alpha: 5
      beta: 10
      ohem_ratio: 3
  - DistillationDMLLoss:
      model_name_pairs:
      - Student
      - Student2
      maps_name: thrink_maps
      weight: 1.0
      key: maps
  - DistillationDBLoss:
      weight: 1.0
      model_name_list:
      - Student
      - Student2
      balance_loss: true
      main_loss_type: DiceLoss
      alpha: 5
      beta: 10
      ohem_ratio: 3
Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Cosine
    learning_rate: 0.001
    warmup_epoch: 2
  regularizer:
    name: L2
    factor: 5.0e-05
PostProcess:
  name: DistillationDBPostProcess
  model_name:
  - Student
  key: head_out
  thresh: 0.3
  box_thresh: 0.6
  max_candidates: 1000
  unclip_ratio: 1.5
Metric:
  name: DistillationMetric
  base_metric_name: DetMetric
  main_indicator: hmean
  key: Student
Train:
  dataset:
    name: SimpleDataSet
    data_dir: ./datasets/icdar2015
    label_file_list:
      - ./datasets/icdar2015/train_label.txt
    ratio_list: [1.0]
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - DetLabelEncode: null
    - IaaAugment:
        augmenter_args:
        - type: Fliplr
          args:
            p: 0.5
        - type: Affine
          args:
            rotate:
            - -10
            - 10
        - type: Resize
          args:
            size:
            - 0.5
            - 3
    - EastRandomCropData:
        size:
        - 640
        - 640
        max_tries: 50
        keep_ratio: true
    - MakeBorderMap:
        shrink_ratio: 0.4
        thresh_min: 0.3
        thresh_max: 0.7
        total_epoch: 500
    - MakeShrinkMap:
        shrink_ratio: 0.4
        min_text_size: 8
        total_epoch: 500
    - NormalizeImage:
        scale: 1./255.
        mean:
        - 0.485
        - 0.456
        - 0.406
        std:
        - 0.229
        - 0.224
        - 0.225
        order: hwc
    - ToCHWImage: null
    - KeepKeys:
        keep_keys:
        - image
        - threshold_map
        - threshold_mask
        - shrink_map
        - shrink_mask
  loader:
    shuffle: true
    drop_last: false
    batch_size_per_card: 8
    num_workers: 4
Eval:
  dataset:
    name: SimpleDataSet
    data_dir: ./datasets/icdar2015
    label_file_list:
      - ./datasets/icdar2015/valid_label.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - DetLabelEncode: null
    - DetResizeForTest: 
        limit_side_len: 960
        limit_type: max
    - NormalizeImage:
        scale: 1./255.
        mean:
        - 0.485
        - 0.456
        - 0.406
        std:
        - 0.229
        - 0.224
        - 0.225
        order: hwc
    - ToCHWImage: null
    - KeepKeys:
        keep_keys:
        - image
        - shape
        - polys
        - ignore_tags
  loader:
    shuffle: false
    drop_last: false
    batch_size_per_card: 1

### 🏃‍♂️ Environment (运行环境)

python 3.8
paddlepaddle-gpu 2.5.2

### 🌰 Minimal Reproducible Example (最小可复现问题的Demo)

python tools/train.py -c configs/det/ch_PP-OCRv4_det_cml.yml

**Comment by jjkgsd** [comment]
> ### 🔎 Search before asking
> * [x]  I have searched the PaddleOCR [Docs](https://paddlepaddle.github.io/PaddleOCR/) and found no similar bug report.[x]  I have searched the PaddleOCR [Issues](https://github.com/PaddlePaddle/PaddleOCR/issues) and found no similar bug report.[x]  I have searched the PaddleOCR [Discussions](https://github.com/PaddlePaddle/PaddleOCR/discussions) and found no similar bug report.
> 
> ### 🐛 Bug (问题描述)
> 使用ch_PP-OCRv4_det_cml.yml知识蒸馏训练，数据集和配置文件都和官方的一样，但是训练准确率和召回率很低，最高只有30%，是什么原因呢？ 

![Image](https://github.com/user-attachments/assets/bedb7446-e61e-409a-9a05-ae5605e9b3af)

配置文件： Global: debug: false use_gpu: true epoch_num: 500 log_smooth_window: 20 print_batch_step: 20 save_model_dir: ./output/ch_ppocr_v4_det_icdar_cml save_epoch_step: 10 eval_batch_step:
> 
> * 0
> * 500
>   cal_metric_during_train: false
>   checkpoints: null
>   pretrained_model:
>   save_inference_dir: ./export_models/ch_ppocr_v4_det_cml
>   use_visualdl: false
>   infer_img: doc/imgs_en/img_10.jpg
>   save_res_path: ./checkpoints/det_db/predicts_db.txt
>   distributed: true
>   Architecture:
>   name: DistillationModel
>   algorithm: Distillation
>   model_type: det
>   Models:
>   Student:
>   pretrained:
>   model_type: det
>   algorithm: DB
>   Transform: null
>   Backbone:
>   name: PPLCNetV3
>   scale: 0.75
>   pretrained: false
>   det: true
>   Neck:
>   name: RSEFPN
>   out_channels: 96
>   shortcut: true
>   Head:
>   name: DBHead
>   k: 50
>   Student2:
>   pretrained:
>   model_type: det
>   algorithm: DB
>   Transform: null
>   Backbone:
>   name: PPLCNetV3
>   scale: 0.75
>   pretrained: true
>   det: true
>   Neck:
>   name: RSEFPN
>   out_channels: 96
>   shortcut: true
>   Head:
>   name: DBHead
>   k: 50
>   Teacher:
>   pretrained: ./pretrained_models/teacher.pdparams
>   freeze_params: true
>   return_all_feats: false
>   model_type: det
>   algorithm: DB
>   Backbone:
>   name: ResNet_vd
>   in_channels: 3
>   layers: 50
>   Neck:
>   name: LKPAN
>   out_channels: 256
>   Head:
>   name: DBHead
>   kernel_list:
>   - 7
>   - 2
>   - 2
>   k: 50
>   Loss:
>   name: CombinedLoss
>   loss_config_list:
> * DistillationDilaDBLoss:
>   weight: 1.0
>   model_name_pairs:
>   
>   * * Student
>     * Teacher
>   * * Student2
>     * Teacher
>       key: maps
>       balance_loss: true
>       main_loss_type: DiceLoss
>       alpha: 5
>       beta: 10
>       ohem_ratio: 3
> * DistillationDMLLoss:
>   model_name_pairs:
>   
>   * Student
>   * Student2
>     maps_name: thrink_maps
>     weight: 1.0
>     key: maps
> * DistillationDBLoss:
>   weight: 1.0
>   model_name_list:
>   
>   * Student
>   * Student2
>     balance_loss: true
>     main_loss_type: DiceLoss
>     alpha: 5
>     beta: 10
>     ohem_ratio: 3
>     Optimizer:
>     name: Adam
>     beta1: 0.9
>     beta2: 0.999
>     lr:
>     name: Cosine
>     learning_rate: 0.001
>     warmup_epoch: 2
>     regularizer:
>     name: L2
>     factor: 5.0e-05
>     PostProcess:
>     name: DistillationDBPostProcess
>     model_name:
> * Student
>   key: head_out
>   thresh: 0.3
>   box_thresh: 0.6
>   max_candidates: 1000
>   unclip_ratio: 1.5
>   Metric:
>   name: DistillationMetric
>   base_metric_name: DetMetric
>   main_indicator: hmean
>   key: Student
>   Train:
>   dataset:
>   name: SimpleDataSet
>   data_dir: ./datasets/icdar2015
>   label_file_list:
>   
>   * ./datasets/icdar2015/train_label.txt
>     ratio_list: [1.0]
>     transforms:
>   * DecodeImage:
>     img_mode: BGR
>     channel_first: false
>   * DetLabelEncode: null
>   * IaaAugment:
>     augmenter_args:
>     
>     * type: Fliplr
>       args:
>       p: 0.5
>     * type: Affine
>       args:
>       rotate:
>       
>       * -10
>       * 10
>     * type: Resize
>       args:
>       size:
>       
>       * 0.5
>       * 3
>   * EastRandomCropData:
>     size:
>     
>     * 640
>     * 640
>       max_tries: 50
>       keep_ratio: true
>   * MakeBorderMap:
>     shrink_ratio: 0.4
>     thresh_min: 0.3
>     thresh_max: 0.7
>     total_epoch: 500
>   * MakeShrinkMap:
>     shrink_ratio: 0.4
>     min_text_size: 8
>     total_epoch: 500
>   * NormalizeImage:
>     scale: 1./255.
>     mean:
>     
>     * 0.485
>     * 0.456
>     * 0.406
>       std:
>     * 0.229
>     * 0.224
>     * 0.225
>       order: hwc
>   * ToCHWImage: null
>   * KeepKeys:
>     keep_keys:
>     
>     * image
>     * threshold_map
>     * threshold_mask
>     * shrink_map
>     * shrink_mask
>       loader:
>       shuffle: true
>       drop_last: false
>       batch_size_per_card: 8
>       num_workers: 4
>       Eval:
>       dataset:
>       name: SimpleDataSet
>       data_dir: ./datasets/icdar2015
>       label_file_list:
>     * ./datasets/icdar2015/valid_label.txt
>       transforms:
>   * DecodeImage:
>     img_mode: BGR
>     channel_first: false
>   * DetLabelEncode: null
>   * DetResizeForTest:
>     limit_side_len: 960
>     limit_type: max
>   * NormalizeImage:
>     scale: 1./255.
>     mean:
>     
>     * 0.485
>     * 0.456
>     * 0.406
>       std:
>     * 0.229
>     * 0.224
>     * 0.225
>       order: hwc
>   * ToCHWImage: null
>   * KeepKeys:
>     keep_keys:
>     
>     * image
>     * shape
>     * polys
>     * ignore_tags
>       loader:
>       shuffle: false
>       drop_last: false
>       batch_size_per_card: 1
> 
> ### 🏃‍♂️ Environment (运行环境)
> python 3.8 paddlepaddle-gpu 2.5.2
> 
> ### 🌰 Minimal Reproducible Example (最小可复现问题的Demo)
> python tools/train.py -c configs/det/ch_PP-OCRv4_det_cml.yml

**Comment by changdazhou** [comment]
数据量有多少呢

**Comment by jjkgsd** [comment]
> 数据量有多少呢

就是官方的数据集,ICDAR2015,1000训练500验证。

**Comment by changdazhou** [comment]
几卡训练的呢，官方配置是4卡训练的，如果本地单卡训练需要减少学习率为原来的1/4

**Comment by jjkgsd** [comment]
> 几卡训练的呢，官方配置是4卡训练的，如果本地单卡训练需要减少学习率为原来的1/4
用的是本地单卡A100训练的

**Comment by that-learner** [comment]
我是微调训练v4 server版，指标也差不多只有30多，比微调前指标还低，可能v4不好训练

**Comment by lengmouling** [comment]
loss是多少啊？我这边训练会发现loss在10以下一点，但是普通训练server或者普通的loss是正常的0.几左右，用蒸馏loss高的离谱

**Comment by changdazhou** [comment]
V4预训练数据量大，确实不太容易微调哈，另外建议微调是时候学习率尽量小一些

**Comment by Sunting78** [comment]
您好，请问用的哪个分支呢？
